packages/profiling-node/test/cpu_profiler.test.ts

```diff
@@ -19,10 +19,10 @@ const fibonacci = (n: number): number => {
 };
 
 const wait = (ms: number) => new Promise(resolve => setTimeout(resolve, ms));
-const profiled = async (name: string, fn: () => void, format: 0 | 1 = 0) => {
+const profiled = async (name: string, fn: () => void) => {
   CpuProfilerBindings.startProfiling(name);
   await fn();
-  return CpuProfilerBindings.stopProfiling(name, format);
+  return CpuProfilerBindings.stopProfiling(name, ProfileFormat.THREAD);
```

Comment:
This started complaining in TS, no idea why. I fixed it by just inlining the one place that we used a different format.